.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,36 +1,23 @@
 <!--- Please review your changes in preview mode -->
 <!--- Provide a general summary of your changes in the Title above -->
 
-#### Issue-ID (If there is an associated issue)
-<!--- Replace XXXX below (2 places) with the actual JIRA id -->
-* **[PP-XXXX](https://pbspro.atlassian.net/browse/PP-XXXX)**
-
-#### Problem description 
-* Clearly describe the problem as it affects the customer
-* Clearly describe the steps to reproduce the problem
-* If the issue is intermittent and not easily reproducible, describe the situation/scenarios/configurations where it was likely triggered
-* Specify the exact platform where the bug is observed, for example CentOS 7.2
-
-Make it easy to understand and reproduce the problem that the customer sees.  You can include short illustrative excerpts from log files or terminal sessions.  Do not include large sections of log files or terminal sessions.  The description should never be empty.
+#### Bug/feature Description
+* Clearly and concisely describe the problem/feature as it affects the customer (this cannot be empty)
+* Bugs: clearly describe the steps to easily reproduce the problem; you can include short illustrative excerpts from log files or terminal sessions
+* Bugs: if the issue is intermittent and not easily reproducible, describe the situation/scenarios/configurations where it was likely triggered
+* If there is an issue ID, link to it: [PP-XXXX](https://pbspro.atlassian.net/browse/PP-XXXX)
 
 #### Affected Platform(s) and PBS Version
-* Specify the exact platform where the bug is observed or the feature is supported, for example CentOS 7.2
-* For bugs, specify the version where the the bug is observed
-
-#### Cause / Analysis
-* Briefly describe root cause/analysis of the problem
+* Platform (for example CentOS 7.2)
+* Bugs: version where observed (for example PBS 14.2.1)
 
-#### Solution description
-* Please describe your changes in detail for reviewers
-
-#### External design (if one exists)
+#### Cause / Analysis / Design
 * Add a link to your external design **[project documentation area](https://pbspro.atlassian.net/wiki/display/PD)**
-
-#### Link to Forum discussion (if it exists)
 * Add a link to the discussion on the Forum **[pbspro community forum](http://community.pbspro.org/)**
```

Comment:
Now that you've removed the headers to these two bullets, you've removed the text "if one exists"
It makes it sound like they are required for all check-ins.  I personally think the header comments are fine, but if you want to get rid of them, please make it clear that the external design and forum link are not required for all checkins.

Reply:
I am fine with the changes i requested. Please address @bhroam's comment. I approve from my side.

Reply:
Done.